public/app/plugins/datasource/cloud-monitoring/components/QueryEditor.tsx

```diff
@@ -1,21 +1,16 @@
-import { css } from '@emotion/css';
 import React, { PureComponent } from 'react';
 
 import { QueryEditorProps, toOption } from '@grafana/data';
-import { config } from '@grafana/runtime';
-import { Button, EditorRows, Select } from '@grafana/ui';
+import { EditorRows } from '@grafana/ui';
 
-import { QUERY_TYPES, SELECT_WIDTH } from '../constants';
 import CloudMonitoringDatasource from '../datasource';
-import { CloudMonitoringQuery, EditorMode, MetricQuery, QueryType, SLOQuery, CloudMonitoringOptions } from '../types';
+import { CloudMonitoringQuery, MetricQuery, QueryType, SLOQuery, CloudMonitoringOptions } from '../types';
 
-import { MetricQueryEditor as ExperimentalMetricQueryEditor } from './Experimental/MetricQueryEditor';
-import { QueryHeader } from './Experimental/QueryHeader';
-import { SLOQueryEditor as ExperimentalSLOQueryEditor } from './Experimental/SLOQueryEditor';
 import { defaultQuery } from './MetricQueryEditor';
+import { QueryHeader } from './QueryHeader';
 import { defaultQuery as defaultSLOQuery } from './SLO/SLOQueryEditor';
```

Comment:
you need to change this import to `from './SLOQueryEditor'`

Reply:
Yep, of course :bow: